src/renderer/features/multisig-wallet/index.tsx

```diff
@@ -170,21 +170,8 @@ transactionSDK(multisigWalletFeature, {
       const extrinsic = transactionService.createExtrinsic(transaction, api);
 
       return transactionService.getExtrinsicWeight(extrinsic).then(maxWeight => {
-        const multisigTransaction: MultisigTransaction = {
-          type: 'decoded',
-          section: 'multisig',
-          method: 'asMulti',
-          args: {
-            threshold: account.threshold,
-            otherSignatories,
-            maybeTimepoint: null,
-            call: encodedTransaction.callData,
-            maxWeight,
-          },
-        };
-
         if (accountUtils.isFlexibleMultisigAccount(account)) {
```

Comment:
The branching in this function looks quite complex due to an attempt to combine flex and non flex flow into a single path. I suggest to explicitly separate them so you dont need to maintain "this path is only for flex" in your head while reading. It willl result in more code but will be easier to understand. Plus this function is big already and needs to be split